github.js

```diff
@@ -458,6 +458,26 @@
             });
          };
 
+         // Merge a specific pull request
+         // -------
+
+         this.mergePull = function(pull, message, cb) {
```

Comment:
Since we're using already an object as the first parameter, instead of using  overloading I think it's better to add the optional message as a property of the object. If it isn't defined we can use a default message (e.g. empty string).